ssh.bash

```diff
@@ -15,21 +15,36 @@ ssh_read_data() {
   read -r ssh_password < <(echo "$contents")
 
   while read -r line; do
-  if [[ "$line" == ssh:* ]]; then
-    local ssh_address=${line#"ssh:"}
-    ssh_address=${ssh_address##+([[:space:]])}
-  fi
-  if [[ "$line" == sshflags:* ]]; then
-    local ssh_flags=${line#"sshflags:"}
-    ssh_flags=${ssh_flags##+([[:space:]])}
-  fi
+    if [[ "$line" == ssh:* ]]; then
+      local ssh_address=${line#"ssh:"}
+      ssh_address=${ssh_address##+([[:space:]])}
+    fi
+    if [[ "$line" == sshflags:* ]]; then
+      local ssh_flags=${line#"sshflags:"}
+      ssh_flags=${ssh_flags##+([[:space:]])}
+    fi
+    if [[ "$line" == user:* ]]; then
+      local ssh_user=${line#"user:"}
+#      ssh_user=${ssh_user##+([[:space::]])}
+    fi
+    if [[ "$line": == IP:* ]]; then
+      local ssh_host=${line#"IP:"}
+#      ssh_host=${ssh_host##+([[:space::]])}
+    fi
   done < <(echo "$contents")
+  if [[ -z ${ssh_address+x} ]]; then
+    ssh_address="${ssh_user// /}@${ssh_host// /}"
+  fi
+
   [[ -z ${ssh_address+x} ]] && die "$path: SSH host not defined."
   # Default value for flags
   ssh_flags="${ssh_flags:-}"
 
+  if [[ "$TERM" == "xterm-kitty" ]]; then
```

Comment:
What does this do? Shouldn't this go into your .bashrc / .zshrc?

Reply:
Sorry this wasn't supposed to be in the merge request. This is a personal setting, putting it in .bashrc / .zshrc breaks some other things in my setup. It's the first time I'm trying to do a pull request to a public repo, I guess I will make a branch and make a pull request from there. Sorry for wasting your time. Please cancel this request, I'll try to do it properly next time.